public/js/selfoss-base.js

```diff
@@ -20,13 +20,18 @@ var selfoss = {
         source: '',
         ajax: true
     },
-
+    
```

Comment:
What gives you the idea, that there should be no whitespaces? Do you really want to discuss such trivia? I don't want to. So, anyway...